### PR TITLE
Add io.github.OpenToonz

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

File: io.github.OpenToonz.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+modules:     
+  - name: gfortran
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/gfortran-62/install.sh
+
+  - name: glu
```
Comment (nedrichards) [inline io.github.OpenToonz.yaml]: should use shared-module https://github.com/flathub/shared-modules/

Comment (nedrichards) [reply io.github.OpenToonz.yaml]: Oh, I can see the issue here of json vs yaml. Interesting.

**Comment by nedrichards** [comment]
bot, build io.github.OpenToonz

**Comment by nedrichards** [comment]
Do you plan to upstream the desktop file and appdata? That'd be very useful for the project.

File: io.github.OpenToonz.yaml (hunk 1 above)
Comment (eszlari) [reply io.github.OpenToonz.yaml]: Actually, I tried shared-module with another app, but I think the added complexity is not worth it.

**Comment by eszlari** [comment]
> Do you plan to upstream the desktop file and appdata? That'd be very useful for the project.

Of course (as I have always done).

File: io.github.OpenToonz.yaml (hunk 1 above)
Comment (barthalion) [reply io.github.OpenToonz.yaml]: What is exactly the problem and what complexity are we talking about?

File: io.github.OpenToonz.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      - FC=gfortran
+      - NO_LAPACKE=1
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/xianyi/OpenBLAS/archive/v0.3.0.tar.gz
+        sha256: cf51543709abe364d8ecfb5c09a2b533d2b725ea1a66f203509b21a8e9d8f1a1
+
+  - name: boost
```
Comment (barthalion) [inline io.github.OpenToonz.yaml]: Would be better to limit it to libraries it needs.

**Comment by barthalion** [comment]
Hi @masafumi-inoue and @shun-iwasawa, will you be interested in having push access to the repository? We want to make sure that upstream has it, even just for the sake of authors having possibility to make changes.

**Comment by nedrichards** [comment]
bot, build io.github.OpenToonz

**Comment by nedrichards** [comment]
bot, build io.github.OpenToonz

File: io.github.OpenToonz.yaml (hunk 1 above)
Comment (nedrichards) [reply io.github.OpenToonz.yaml]: Aside from this discussion, lgtm

**Comment by barthalion** [comment]
Builds fine. What is the issue with glu from shared-modules?

**Comment by eszlari** [comment]
> What is the issue with glu from shared-modules?

There's no issue, I just think it's easier for maintenance to have all modules in a single file manifest.

**Comment by barthalion** [comment]
I don't think it increases maintenance effort. The main benefit from using it from shared-modules, apart from consistency with other manifests on Flathub, is that we ensure files are deduplicated correctly as module is identical everywhere.

**Comment by eszlari** [comment]
> we ensure files are deduplicated correctly as module is identical everywhere

Okay, this is a good reason. How about copying the module into the git repo?

**Comment by barthalion** [comment]
It complicated updating though, with submodule you just need to run `git submodule update --remote --merge` and commit changes.

**Pull request closed without merging**